sinon/lib/util/Wrapper.py

```diff
@@ -2,6 +2,11 @@
 Copyright (c) 2016-2017, Kir Chou
 https://github.com/note35/sinon/blob/master/LICENSE
 """
+import traceback
+import sys
+sys.path.insert(0, '../../')
```

Comment:
This is a bad idea. Please read my comment [here](https://github.com/note35/sinon/pull/8#issuecomment-334007839)